examples/hello.rb

```diff
@@ -0,0 +1,16 @@
+$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
+
```

Comment:
This should be removed before merging; it should not be needed assuming the gem is installed.